instrumentation/otel-webserver-module/src/core/sdkwrapper/SdkHelperFactory.cpp

```diff
@@ -96,9 +96,17 @@ SdkHelperFactory::SdkHelperFactory(
         " and LibraryVersion " << libraryVersion);
 
     // Adding trace propagator
-    using MapHttpTraceCtx = opentelemetry::trace::propagation::HttpTraceContext;
-    mPropagators.push_back(
-        std::unique_ptr<MapHttpTraceCtx>(new MapHttpTraceCtx()));
+    if(config->getOtelPropagatorType() == "b3"){
+        mPropagators.push_back(
+            std::unique_ptr<opentelemetry::trace::propagation::B3PropagatorMultiHeader>(
+                new opentelemetry::trace::propagation::B3PropagatorMultiHeader()));
+    }
+    else{
+        mPropagators.push_back(
+            std::unique_ptr<opentelemetry::trace::propagation::HttpTraceContext>(
+                new opentelemetry::trace::propagation::HttpTraceContext()));
+    }
+    
```

Comment:
Is that a fair understanding that at a time only one propagator can exist. Co-existence of both propagators is not supported?

Reply:
I could implement it in a co-existing way as well, but i think its better to only have one propagator at a time so that we have expected behaviour all the times.